src/quacc/recipes/newtonnet/ts.py

```diff
@@ -266,6 +282,269 @@ def quasi_irc_job(
     return relax_summary
 
 
+@job
+@requires(
+    has_newtonnet, "NewtonNet must be installed. Refer to the quacc documentation."
+)
+@requires(
+    has_geodesic_interpolate,
+    "geodesic-interpolate must be installed. Refer to the quacc documentation.",
+)
+def neb_job(
+    reactant_atoms: Atoms,
+    product_atoms: Atoms,
+    relax_job_kwargs: dict[str, Any] | None = None,
+    calc_kwargs: dict[str, Any] | None = None,
+    geodesic_interpolate_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+) -> NebSchema:
```

Comment:
This comment applies to both recipes:

Like we did with sella and optimizations, we are going to want all the recipes here to be run without geodesic-interpolate if desired. geodesic-interpolate is merely one of many interpolation methods, and these functions do not strictly rely on geodesic-interpolate for functionality --- they are only relying on it to generate a set of images. Many methods can do that. The main concern is that one day, geodesic-interpolate _will_ stop working and break (because it's not actively maintained). We want to make sure the recipes can still be used if/when that happens, or even in cases where better methods come along.

To do this, I would suggest:
- Removing the `@requires` decorator for geodesic-interpolate
- Having a keyword argument that's like `interpolation_method: Literal["linear", "geodesic"] = "linear"`. If set to `"geodesic"`, then call your geodesic-interpolate wrapper. If set to `"linear"`, use ASE's default interpolate method. One can easily see how this could be expanded later to include idpp or others.
- Change `geodesic_interpolate_kwargs` to `interpolation_kwargs`, which would be the kwargs passed to whatever interpolation method is used (i.e. geodesic's or ASE's).

With this approach, it will be more flexible. Imagine one day you make a new image interpolation scheme `anup_image_gen`. We could then add it to `interpolation_method: Literal["linear", "geodesic", "anup"]` without having to make a new recipe.